packages/x-charts/src/themeAugmentation/themeAugmentation.spec.ts

```diff
@@ -188,5 +188,53 @@ createTheme({
       //   constent: { color: 'red' },
       // },
     },
+    MuiPieArc: {
+      defaultProps: {
+        id: 'toto',
+        // @ts-expect-error invalid MuiMarkElement prop
+        someRandomProp: true,
+      },
+      styleOverrides: {
+        root: { backgroundColor: 'red' },
+        // @ts-expect-error invalid MuiMarkElement class key
+        constent: { color: 'red' },
+      },
+    },
+    MuiPieArcLabel: {
+      defaultProps: {
+        id: 'toto',
+        // @ts-expect-error invalid MuiMarkElement prop
+        someRandomProp: true,
+      },
+      styleOverrides: {
+        root: { backgroundColor: 'red' },
```

Comment:
Only the root class works here as well.
`faded` and `highlighted` do nothing. 🙈